CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -14,6 +14,51 @@ public function install() {
 
   public function uninstall() {
     $this->executeSqlFile('sql/uninstall.sql');
+    $this->uninstallCustomInformation();
+  }
+
+  /**
+   *  Uninstall custom information
+   */
+  private function uninstallCustomInformation() {
+    $this->deleteField('OptionGroup', 'direct_debit_codes');
+    $this->deleteField('OptionGroup', 'direct_debit_originator_number');
+    $this->deleteField('CustomGroup', 'direct_debit_mandate');
+    $this->deleteField('CustomGroup', 'direct_debit_information');
+    $this->deleteField('UFGroup', 'Direct Debit Information', 'title');
+    $this->deleteField('PaymentProcessor', 'OfflineDirectDebit', "payment_processor_type_id");
+    $this->deleteField('PaymentProcessorType', 'OfflineDirectDebit');
+  }
+
+  /**
+   * Function deletes custom field
```

Comment:
a more clear name to this method is probably **deleteEntityRecord** , also please use drupal standard for dockblocks : https://www.drupal.org/docs/develop/standards

also the method description should be more descriptive/


In  other words, I would write this method as following : 

e.g : 

```php
  /**
   * Deletes a record for the specified CiviCRM entity based on
   * the search field and search value.
   *
   * @param string $entityType
   *   The entity type for the record we want to remove (e.g OptionValue, PaymentProcessor ..etc).
   *   it should be a valid Custom or CiviCRM core entity type. 
   * @param string $searchValue
   *   The search value to find the record that we want to remove
   * @param string $searchField
   *   The field that we search the search value against.
   */
  private function deleteEntityRecord($entityType, $searchValue, $searchField = 'name') {
    civicrm_api3($entityType, 'get', [
      $searchField => $searchValue,
      'api.' . $entityType . '.delete' => ['id' => '$value.id'],
    ]);
  }
```